tests/pds/dataservice_test.go

```diff
@@ -370,11 +353,11 @@ var _ = Describe("{ScaleUPDataServices}", func() {
 				})
 
 				Step("Delete the workload generating deployments", func() {
-					if ds.Name == cassandra || ds.Name == postgresql || ds.Name == redis || ds.Name == rabbitmq {
-						if ds.Name == cassandra || ds.Name == postgresql {
+					if Contains(dataServiceDeploymentWorkloads, ds.Name) || Contains(dataServicePodWorkloads, ds.Name) {
```

Comment:
why do u need this or condition if you separate if condition for dataServiceDeploymentWorkloads and dataServicePodWorkloads again

Reply:
yeah you are right, I have updated the same